backend/apps/progress/views.py

```
@@ -1,1515 +1,1535 @@
+            data['notes'].append({
+                'id': note.id,
+                'lesson_title': note.lesson.title,
+                'module_title': note.lesson.module.title if note.lesson.module else None,
+                'content': note.content,
+                'tags': note.tags if hasattr(note, 'tags') else [],
+                'created_at': note.created_at.isoformat(),
+                'updated_at': note.updated_at.isoformat() if note.updated_at else None,
+            })
+        
+        filename = f"notes_export_{user.username}_{datetime.now().strftime('%Y%m%d_%H%M%S')}.json"
+        json_content = json.dumps(data, indent=2, ensure_ascii=False)
```

Ruff (0.15.21)

[error] Undefined name `json`

(F821)